less_users/tests/test_views.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from django.urls import reverse
+
+from challenges.models import Category, Challenge
+
```

Comment:
nie testowałaś jeszcze żadnych widoków mutujących dane?

Reply:
widoki mutujące dane to...? mam wrażenie, że wracam na początek mojej drogi :(

Reply:
Takie, które wprowadzają zmiany w danych - dodają punkty, tworzą obiekty, usuwają.